html/script.js

```diff
@@ -9,13 +10,13 @@ const OpenMenu = (data) => {
     $(`.main-wrapper`).fadeIn(0);
 	
 	const formID = data.formID != null ? data.formID : "qb-input-form";
```

Comment:
remove `const` keyword and change to `formID = data.formID != null ? data.formID : "qb-input-form";`